Makefile

```diff
@@ -51,7 +51,7 @@ repl:
 
 test:
 	dune build @src/fmt --auto-promote || true
-	dune exec src/hazeltest/hazeltest.exe -- --regression-dir src/hazeltest/regressions
+	dune exec src/haz3ltest/haz3ltest.exe -- --regression-dir src/haz3ltest/regressions
```

Comment:
can you add a blurb to the README file for how to run tests (I removed it recently from dev since it was out of date)